### ISSUE TITLE
CRAN checks

### ISSUE THREAD

**Issue by fkeck** [ropensci/bold#107]
Hi,
I got an email from CRAN warning me to take action for my package which depends on bold because bold is going to be removed from CRAN due to some policy violations.
Are there plans to update bold to keep it on CRAN?

**Comment by salix-d** [comment]
Hi,

Yes, I do plan to update it. I was told to not resubmit until September
though.
For the record, the policy "violations" were:

   - accidentally sending HTML to the cran email (they ask for plain text)
   because I wanted to show a screenshot of
   https://cran.r-project.org/web/checks/check_results_bold.html not
   showing any error
   - there's a connection issue with some tests which gives sporadic errors
   (I think it's mostly at night)

Hopefully, they don't archive it before the time they gave me.

**Comment by fkeck** [comment]
Thanks for the quick reply and good luck with the resubmission!

**Comment by salix-d** [comment]
You're right, it was. It might be hosted elsewhere (R-Multiverse), although
I'm not sure how quickly this will be done.
If it takes too long, I'll resubmit to CRAN.

**Comment by MatteoCe** [comment]
Is the decision to remove bold from CRAN definitive?
Just to know if all packages depending on it should take action in case it is.

**Comment by salix-d** [comment]
I will contact everyone I know depends on it to let them know the steps for
changing the dependency to Rmultiverse.

In the mean time, I will try to resubmit it to CRAN though because I really
don't like it being nowhere right now. Might be able to do that today, or
by the end of the week.

**Comment by salix-d** [comment]
Wan't even a long fix, but _life_, you know.
It finally has been resubmitted to CRAN! Hopefully, it will be up soon.

**Comment by willgearty** [comment]
Any progress on this? It appears that other packages have been bumped off of CRAN (e.g., [taxize](https://github.com/ropensci/taxize/issues/938), [metacoder](https://github.com/grunwaldlab/metacoder/issues/368)) because of their dependency on `bold`.

**Comment by salix-d** [comment]
I had resumbmitted but there were still errors. Even though I could just skip all tests on cran, the thing is, some of these errors are due to their v4 API (the one currentl used by this package) being replaced by their v5 API.

 If you have the time, please see this [discussion](https://github.com/ropensci/bold/discussions/112),  I'd love more input.
